docs/how-to/data-plane-installation.md

```diff
@@ -0,0 +1,72 @@
+# Installing Kuadrant data-plane into an existing OCM Managed Cluster
+
+## Introduction
+This walkthrough will show you how to install and setup the Kuadrant Operator into an [OCM](https://open-cluster-management.io/) [Managed Cluster](https://open-cluster-management.io/concepts/managedcluster/).
+
+## Prerequisites
```

Comment:
This is looking pretty good to me but wondering if we want to link to the install guide now as a pre-req?

Reply:
yep, will add a bi-directional link between them